cpp/client/src/main.cpp

```diff
@@ -0,0 +1,71 @@
+#include "objects.pb.h"
+
+#include "utils/Configuration.h"
+#include "logger/logger.h"
+#include "logger/writer.h"
+#include "logger/formatter.h"
+
+#include "channel/ChannelFactory.h"
+#include "sessions/SessionsClient.h"
+#include "tasks/TasksClient.h"
+#include "results/ResultsClient.h" 
+#include "events/EventsClient.h" 
+
+#include <map>
+
+
+namespace ak_common = armonik::api::common;
+namespace ak_client = armonik::api::client;
+namespace ak_grpc = armonik::api::grpc::v1;
+
+int main() {
+    ak_common::logger::Logger logger{ak_common::logger::writer_console(), ak_common::logger::formatter_plain(true)};
+    ak_common::utils::Configuration config;
+
+    config.add_json_configuration("/appsettings.json").add_env_configuration();
+    logger.info("Initialized client config.");
+    ak_client::ChannelFactory channelFactory(config, logger);
+    std::shared_ptr<::grpc::Channel> channel = channelFactory.create_channel();
+
+    ak_grpc::TaskOptions taskOptions;
+
+    std::string used_partition = "default";
+    logger.info("Using the '"+ used_partition + "' partition.");
+
+    taskOptions.mutable_max_duration()->set_seconds(3600);
+    taskOptions.set_max_retries(3);
+    taskOptions.set_priority(1);
+    taskOptions.set_partition_id(used_partition);
+    taskOptions.set_application_name("hello-cpp");
+    taskOptions.set_application_version("1.0");
+    taskOptions.set_application_namespace("hello-namespace");
```

Comment:
These options are not required when developing native ArmoniK applications so they can be unset.

Reply:
I'm changing the namespace to samples. I think it's better for them to be set. It poses no harm, and it shows how you can set these fields to a potential new user. They might not be required but it should probably be established as a good practice and adopted in the samples. 

The samples also exist to help onboard new users. If none of the samples used these fields, I could see that causing problems for the users. (They might for one think that the application/image name and tag is sufficient)